### PR TITLE
Apply void_checks lint

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: packages/flutter_tools/lib/src/build_runner/web_fs.dart [hunk 1]
```diff
@@ -629,7 +629,6 @@ class BuildDaemonCreator {
         if (testTargets.hasBuildFilters) {
           b.buildFilters.addAll(testTargets.buildFilters);
         }
-        return b;
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/build_runner/web_fs.dart]: @yjbanov this is the only change here that I'm not confident in. I couldn't quickly figure out what happened to this return value.

File: packages/flutter/test/widgets/shortcuts_test.dart [hunk 2]
```diff
@@ -257,7 +255,6 @@ void main() {
                 TestAction.key: () => TestAction(
                   onInvoke: (FocusNode node, Intent intent) {
                     invoked = true;
-                    return true;
                   },
```
Comment (Hixie) [inline packages/flutter/test/widgets/shortcuts_test.dart]: @gspencergoog FYI (in case you were expecting these return values to have any effect)

**Comment by Hixie** [comment]
cc @a14n

File: packages/flutter/test/foundation/diagnostics_test.dart [hunk 3]
```diff
@@ -1914,9 +1914,11 @@ void main() {
             '--- example property at max length --',
             style: propertyStyle,
           ),
-          DiagnosticsProperty<void>(null,
-              'Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap.',
-              allowWrap: false),
+          DiagnosticsProperty<void>(
+            'Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap.',
+            null,
+            allowWrap: false
```
Comment (a14n) [inline packages/flutter/test/foundation/diagnostics_test.dart]: nit: trailing coma

File: packages/flutter_tools/lib/src/build_runner/web_fs.dart (hunk 1 above)
Comment (yjbanov) [reply packages/flutter_tools/lib/src/build_runner/web_fs.dart]: I only rewrote it from the `=>` notation to the `{}` block notation and automatically kept the existing semantics. I didn't check that `DefaultBuildTarget` took a `void` function. So your change looks fine.

File: packages/flutter/test/widgets/shortcuts_test.dart (hunk 2 above)
Comment (gspencergoog) [reply packages/flutter/test/widgets/shortcuts_test.dart]: No, that's just a mistake caught by the lint.

File: packages/flutter/test/foundation/diagnostics_test.dart [hunk 4]
```diff
@@ -1914,9 +1914,11 @@ void main() {
             '--- example property at max length --',
             style: propertyStyle,
           ),
-          DiagnosticsProperty<void>(null,
-              'Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap.',
-              allowWrap: false),
```
Comment (a14n) [inline packages/flutter/test/foundation/diagnostics_test.dart]: Regarding the failing test the fix should be to replace `<void>` with `<String>`

**Review by dnfield — APPROVED** [review]
LGTM

**Review by a14n — APPROVED** [review]
LGTM

**Pull request merged**